app/controllers/presentations_controller.rb

```diff
@@ -11,6 +11,7 @@ def show
 
   def new
     @presentation = Presentation.new
+    @event = Event.find(params[:event_id]) if params[:event_id]
```

Comment:
This comment is showing up as not updated but if you view the diff it actually has been changed.